src/pages/uxp/reference-js/Modules/FileSystem.md

```diff
@@ -165,7 +165,9 @@ await fs.close(fd);
 ## read(fd, buffer, offset, length, position, callback)
 Reads data in chunks from the file it refers to the file descriptor
 
+**Returns**: `Promise<Object>` - { bytesRead, buffer }  
```

Comment:
What types do `bytesRead` and `buffer` have? I assume it's `ArrayBuffer` for the latter, but what is `bytesRead`?

Reply:
@pklaschka Please, refer to the Returns in https://nodejs.org/api/fs.html#filehandlereadbuffer-offset-length-position
`bytesRead` is number which is the number of bytes read, and `buffer` is ArrayBuffer in UXP.

Reply:
Added